recipes/jeepney/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "jeepney" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "a6f2aa72e61660248d4d524dfccb6405f17c693b69af5d60dd7f2bab807d907e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
```

Comment:
Would it be possible to have this as a version constraint on python? Then could this be noarch?